localstack/ext/java/src/main/java/cloud/localstack/TestUtils.java

```diff
@@ -43,12 +47,57 @@ public static AmazonSQS getClientSQS() {
                 withCredentials(getCredentialsProvider()).build();
     }
 
+    public static AmazonSQSAsync getClientSQSAsync() {
```

Comment:
nitpick: can we change this method to:
```
    public static AmazonSQSAsync getClientSQSAsync() {
        return getClientSQSAsync(null);
    }
```
This way we can avoid code duplication and only maintain the logic in a single method. (same for `getClientSNSAsync`, `getClientLambdaAsync`, and `getClientKinesisAsync`)